### ISSUE TITLE
Investigate a webExtension compatible way to do attachment drag and drop (and also allow dnd of files to finder/desktop on Mac)

### ISSUE THREAD

**Issue by Standard8** [protz/thunderbird-conversations#1451]
In porting the drag and drop code across, I noticed that it is highly dependent on constructs from within Thunderbird. I think for it to work for WebExtensions, we'll need to rework it so that we're streaming the attachment to the data transfer, or providing the full information there.

This is also likely to get drag and drop of attachments to finder/desktop working on Mac.

**Comment by DerXteMensch** [comment]
Drag and drop of attachments using Conversations is working for me (for some weeks now)

**Comment by Standard8** [comment]
Yes, it is generally working, however there's more work we need to do to prepare for supporting future versions of Thunderbird, which is what this issue is about.
